$OpenBSD: patch-uigtk2_ml,v 1.1 2005/05/30 19:27:45 sturm Exp $
Post-release developer patch.
o Disable password prompting code (broken in 2.12.0).
  See http://lists.seas.upenn.edu/pipermail/unison-hackers/2005-May/000091.html
--- uigtk2.ml.orig	Sat Mar 12 11:17:59 2005
+++ uigtk2.ml	Fri May 27 17:30:45 2005
@@ -745,8 +745,7 @@ let termInteract() =
     match Uicommon.sshParse s with
       Uicommon.Password account -> getPassword account
     | _ -> "" in
-  if Osx.isMacOSX or Osx.isLinux then Some handleSSH
-  else None
+  None
 
 (* ------ *)
 
